server/http.go

```diff
@@ -195,7 +201,8 @@ func (p *Plugin) serveHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 		return p.httpChannelSubscriptions(w, r)
 	}
 
-	return respondErr(w, http.StatusNotFound, errors.New("not found"))
+	p.gorillaRouter.ServeHTTP(w, r)
+	return 0, nil
```

Comment:
Should `p.gorillaRouter` handle the 404 case?

Reply:
It already does, doesn't it? If this plugin is to move forward, 2/5 we should probably convert it to use gorilla anyway, I just didn't want to do it as part of this PR, it's a little crafty.

Reply:
I see. The code could be cleaner, but given that there are plans to migrate to gorilla, it's fine. Is there a HW yet?